controllers/articles.controllers.js

```diff
@@ -0,0 +1,109 @@
+const fetchArticleById = require('../models/fetchArticleById.models.js')
+exports.getArticleByID= async (req,res,next)=>{
+    const articleID = Object.values(req.params).toString()
```

Comment:
Seems like a long way to get the article id! We can use` const {article_id} = req.params` to destructure it from the object